cl/people_db/api_serializers.py

```diff
@@ -121,9 +121,9 @@ def get_disclosure_years(self, obj: Person) -> str:
     def get_thumbnail_path(self, obj: Person) -> str:
         return make_person_picture_path(obj)
 
-    def get_oldest_disclosure_url(self, obj: Person) -> str:
+    def get_newest_disclosure_url(self, obj: Person) -> str:
         """Get the URL of the"""
-        return obj.disclosures[-1].get_absolute_url()
```

Comment:
Wow, normally you can't use negative indexing on a queryset, but I guess because it's prefetched you can. That's a very big surprise, but cool, cool, it works, thanks.